File: Hydrogene/Toffee/OS X/NSTableView/NSTableView/Person.cs
using AppKit;

namespace NSTableView
{
    public class Person
    {
        private string fName;
        private NSInteger fAge;

        public string name {
            get {
              return fName;
             }
             set {
               fName = value;
             }
        }

        public NSInteger age {
            get {
              return fAge;
             }
             set {
               fAge = value;
             }
        }		
    }
}

File: Oxygene/Toffee/OS X/NSTableView/NSTableView/Person.pas
namespace NSTableView;

interface

uses
  Foundation;

type
  Person = public class
  private
    fName: String;
    fAge: Integer;
  protected
  public
    property name : String read fName write fName;
    property age : Integer read fAge write fAge;
  end;

implementation

end.
